examples/walkthrough/README.md

```diff
@@ -1,74 +1,167 @@
-# Kubernetes 101 - Walkthrough
+# Kubernetes 101 - Kubectl CLI & Pods
+
+Table of Contents
+- [Kubectl CLI](#kubectl-cli)
+  - [Install Kubectl](#install-kubectl)
+  - [Configure Kubectl](#configure-kubectl)
+- [Pods](#pods)
+  - [Pod Definition](#pod-definition)
+  - [Pod Management](#pod-management)
+  - [Volumes](#volumes)
+  - [Multiple Containers](#multiple-containers)
+- [What's Next?](#whats-next)
+
+
+## Kubectl CLI
+
+The easiest way to interact with Kubernetes is via the command-line interface.
+
+If you downloaded a pre-compiled release, kubectl should be under `platforms/<os>/<arch>`.
+
+If you built from source, kubectl should be either under `_output/local/bin/<os>/<arch>` or `_output/dockerized/bin/<os>/<arch>`.
+
+For more info about kubectl, including its usage, commands, and parameters, see the [kubectl CLI reference](/docs/kubectl.md).
+
+#### Install Kubectl
+
+The kubectl binary can be executed from anywhere, but the rest of the walkthrough will assume that it's in your PATH.
```

Comment:
Sentence does not really make sense. If it is in the path, it can be run from anywhere. The "but" is strange.